app/src/main/java/org/wikipedia/createaccount/CreateAccountActivity.kt

```diff
@@ -65,8 +65,11 @@ class CreateAccountActivity : BaseActivity() {
     }
 
     private fun setClickListeners() {
-        binding.viewCreateAccountError.backClickListener = View.OnClickListener { onBackPressed() }
-        binding.viewCreateAccountError.retryClickListener = View.OnClickListener { it.visibility = View.GONE }
+        binding.viewCreateAccountError.backClickListener = View.OnClickListener {
+            binding.viewCreateAccountError.visibility = View.GONE
+            onBackPressed()
```

Comment:
It looks like we only need the `binding.viewCreateAccountError.visibility = View.GONE` and don't have to run the `onBackPressed()`, since it will exit the `CreateAccountActivity`.

Reply:
correct